deb-x64/Dockerfile

```diff
@@ -138,6 +140,13 @@ RUN curl -sSfL -o golangci-lint-install.sh https://raw.githubusercontent.com/gol
     && echo "942efc00799b83aebaf6628c34e1151467b7fa21f0cfd78554279dc73f8c8ef3  $GOPATH/bin/golangci-lint" | sha256sum --check \
     && rm golangci-lint-install.sh
 
+# Rust is needed to compile some python libs
+RUN curl -sSL -o rustup-init.sh https://raw.githubusercontent.com/rust-lang/rustup/1.24.3/rustup-init.sh \
+    && echo "a3cb081f88a6789d104518b30d4aa410009cd08c3822a1226991d6cf0442a0f8  rustup-init.sh" | sha256sum --check \
+    && sh rustup-init.sh -y --default-toolchain ${RUST_VERSION} \
+    && echo "3dc5ef50861ee18657f9db2eeb7392f9c2a6c95c90ab41e45ab4ca71476b4338  $HOME/.cargo/bin/rustc" | sha256sum --check
```

Comment:
One small nit: if the `RUST_VERSION` is configurable, then the sha256 should also be, otherwise this check will fail on any `RUST_VERSION` that's not `1.60.0`

Reply:
IMO `RUST_VERSION` is as configurable as the sha as is, but I can move the SHA to the top.

Reply:
I don't think anyone is using the `ARGs` to the Dockerfile anyway and would rather remove them, maybe leaving some things as `ENV` so they can be reused.

Reply:
I see your point. For this PR let's leave everything in a consistent state, and then we can discuss removing all `ARG`s from the Dockerfile.